### PR TITLE
Added sections Getting Help and Opening Issues

### PR DESCRIPTION
**Pull request by Dominitio**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
 
-This is the .NET driver for [Amazon Quantum Ledger Database (QLDB)](https://aws.amazon.com/qldb/), which allows .NET developers
-to write software that makes use of AmazonQLDB.
-
 [![nuget](https://img.shields.io/nuget/v/Amazon.QLDB.Driver.svg)](https://www.nuget.org/packages/Amazon.QLDB.Driver/)
 [![license](https://img.shields.io/badge/license-Apache%202.0-blue)](https://github.com/awslabs/amazon-qldb-driver-dotnet/blob/master/LICENSE)
 [![AWS Provider](https://img.shields.io/badge/provider-AWS-orange?logo=amazon-aws&color=ff9900)](https://aws.amazon.com/qldb/)
 
+This is the .NET driver for [Amazon Quantum Ledger Database (QLDB)](https://aws.amazon.com/qldb/), which allows .NET developers
+to write software that makes use of AmazonQLDB.
```
Comment (byronlin13) [inline README.md:8]:
```suggestion
to write software that makes use of Amazon QLDB.
```

File: README.md [hunk 2]
```diff
@@ -57,6 +56,19 @@ To run the integration tests, you must run it on the command line with the follo
...
+* Open a support ticket with [AWS Support](http://docs.aws.amazon.com/awssupport/latest/user/getting-started.html).
+* If you think you may have found a bug, please open an [issue](https://github.com/awslabs/amazon-qldb-driver-dotnet/issues/new).
+
+## Opening Issues
+
+If you encounter a bug with the Amazon QLDB Dotnet Driver, we would like to hear about it. Please search the [existing issues](https://github.com/awslabs/amazon-qldb-driver-dotnet/issues) and see if others are also experiencing the issue before opening a new issue. When opening a new issue, we will need the version of Amazon QLDB Dotnet Driver, Dotnet language version, and OS you’re using. Please also include reproduction case for the issue when appropriate.
+
+The GitHub issues are intended for bug reports and feature requests. For help and questions with using AWS SDK for GO please make use of the resources listed in the [Getting Help](https://github.com/awslabs/amazon-qldb-driver-dotnet#getting-help) section. Keeping the list of open issues lean will help us respond in a timely manner.
+
 ### Documentation
```
Comment (byronlin13) [inline README.md:72]: documentation is a subsection fo development

File: README.md [hunk 3]
```diff
@@ -57,6 +56,19 @@ To run the integration tests, you must run it on the command line with the follo
...
+## Getting Help
+
+Please use these community resources for getting help.
+* Ask a question on StackOverflow and tag it with the [amazon-qldb](https://stackoverflow.com/questions/tagged/amazon-qldb) tag.
+* Open a support ticket with [AWS Support](http://docs.aws.amazon.com/awssupport/latest/user/getting-started.html).
+* If you think you may have found a bug, please open an [issue](https://github.com/awslabs/amazon-qldb-driver-dotnet/issues/new).
+
+## Opening Issues
+
+If you encounter a bug with the Amazon QLDB Dotnet Driver, we would like to hear about it. Please search the [existing issues](https://github.com/awslabs/amazon-qldb-driver-dotnet/issues) and see if others are also experiencing the issue before opening a new issue. When opening a new issue, we will need the version of Amazon QLDB Dotnet Driver, Dotnet language version, and OS you’re using. Please also include reproduction case for the issue when appropriate.
```
Comment (byronlin13) [inline README.md:68]:
```suggestion
If you encounter a bug with the Amazon QLDB .NET Driver, we would like to hear about it. Please search the [existing issues](https://github.com/awslabs/amazon-qldb-driver-dotnet/issues) and see if others are also experiencing the issue before opening a new issue. When opening a new issue, we will need the version of Amazon QLDB .NET Driver, .NET language version, and OS you’re using. Please also include reproduction case for the issue when appropriate.
```

File: README.md [hunk 4]
```diff
@@ -57,6 +56,19 @@ To run the integration tests, you must run it on the command line with the follo
...
+
+The GitHub issues are intended for bug reports and feature requests. For help and questions with using AWS SDK for GO please make use of the resources listed in the [Getting Help](https://github.com/awslabs/amazon-qldb-driver-dotnet#getting-help) section. Keeping the list of open issues lean will help us respond in a timely manner.
```
Comment (byronlin13) [inline README.md:70]:
```suggestion
The GitHub issues are intended for bug reports and feature requests. For help and questions with using AWS QLDB .NET Driver, please make use of the resources listed in the [Getting Help](https://github.com/awslabs/amazon-qldb-driver-dotnet#getting-help) section. Keeping the list of open issues lean will help us respond in a timely manner.
```

File: README.md (hunk 1 above)
Comment (Dominitio) [reply README.md:8]: Done

File: README.md (hunk 3 above)
Comment (Dominitio) [reply README.md:68]: Done

File: README.md (hunk 4 above)
Comment (Dominitio) [reply README.md:70]: Done

File: README.md (hunk 2 above)
Comment (Dominitio) [reply README.md:72]: Moved

**Comment by byronlin13** [comment]
LGTM, could you rerun the code snippets in the README to make sure they still work?

File: README.md [hunk 5]
```diff
@@ -1,12 +1,11 @@
...
 
-This is the .NET driver for [Amazon Quantum Ledger Database (QLDB)](https://aws.amazon.com/qldb/), which allows .NET developers
-to write software that makes use of AmazonQLDB.
-
 [![nuget](https://img.shields.io/nuget/v/Amazon.QLDB.Driver.svg)](https://www.nuget.org/packages/Amazon.QLDB.Driver/)
 [![license](https://img.shields.io/badge/license-Apache%202.0-blue)](https://github.com/awslabs/amazon-qldb-driver-dotnet/blob/master/LICENSE)
 [![AWS Provider](https://img.shields.io/badge/provider-AWS-orange?logo=amazon-aws&color=ff9900)](https://aws.amazon.com/qldb/)
 
+This is the .NET driver for [Amazon Quantum Ledger Database (QLDB)](https://aws.amazon.com/qldb/), which allows .NET developers
+to write software that makes use of Amazon QLDB.
```
Comment (kylepbit) [inline README.md:9]: Go has "For getting started with the driver, see Go and Amazon QLDB." and a link to the getting started, which seems like it belongs here.

Comment (kylepbit) [reply README.md:9]: The Getting started is missing the code snippet to list tables like Python and Node?

Comment (kylepbit) [reply README.md:9]: The See Also section is missing items vs. the Python version?

File: README.md [hunk 6]
```diff
@@ -65,6 +64,18 @@ You can generate the docstring HTML locally by running the following in the root
 
 ```docfx docs/docfx.json --serve```
 
+## Getting Help
+
+Please use these community resources for getting help.
+* Ask a question on StackOverflow and tag it with the [amazon-qldb](https://stackoverflow.com/questions/tagged/amazon-qldb) tag.
+* Open a support ticket with [AWS Support](http://docs.aws.amazon.com/awssupport/latest/user/getting-started.html).
+* If you think you may have found a bug, please open an [issue](https://github.com/awslabs/amazon-qldb-driver-dotnet/issues/new).
```
Comment (kylepbit) [inline README.md:72]: Missing the entry for the QLDB Forum that's in the Python version.

**Review by kylepbit — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (Dominitio) [reply README.md:9]: Added Dotnet Cookbook and DMV samples; but there's no equivalent for Driver Tutorial.

File: README.md (hunk 6 above)
Comment (Dominitio) [reply README.md:72]: Added

File: README.md (hunk 5 above)
Comment (kylepbit) [reply README.md:9]: @Dominitio I see the listing table snippet still isn't present?

**Pull request closed without merging**